examples/simple_zonal_with_acm/acm.tf

```diff
@@ -25,4 +25,6 @@ module "acm" {
   policy_dir  = "foo-corp"
 
   secret_type = "ssh"
+
+  policy_bundles = "https://github.com/GoogleCloudPlatform/acm-policy-controller-library.git/bundles/policy-essentials-v2022"
```

Comment:
Can we pin these to a released version?

Reply:
Sure, this should pin it to a commit hash